docs/bare-metal/README.md

```diff
@@ -0,0 +1,298 @@
+ * first sector: default
+ * last sector: +50G
+ * Command: w
+
+ * reboot
+
+
+don't put a file system on the partion.
+
+Note that I am creating a partion size 50 Gigs, this is for docker.  Just make it consistent across your cluster.
+
+if you have MS-dos partions make it a primary partion, not an extended partion or it won't work.
+
+Additionally.  If you hang on TASK "lvm | create volume group" then you will have to apply the patch
+https://github.com/ansible/ansible-modules-extras/issues/1504 the file you need to do this to is in
+mantl base directory /library/lvg.py
+
+
+Do all the computers to this point.
+
+## Creating Your Inventory
+
+Here is an example inventory file. It should be place in the root of the mantl directory.
+
+    [role=control]
+    control01 private_ipv4=172.16.222.6 ansible_ssh_host=172.16.222.6
+    control02 private_ipv4=172.16.222.7 ansible_ssh_host=172.16.222.7
+    control03 private_ipv4=172.16.222.8 ansible_ssh_host=172.16.222.8
+
+    [role=control:vars]
+    consul_is_server=true
+    lvm_physical_device=/dev/sda3
+
+    [role=worker]
+    resource01 private_ipv4=172.16.222.11 ansible_ssh_host=172.16.222.11
+    resource02 private_ipv4=172.16.222.12 ansible_ssh_host=172.16.222.12
+    resource03 private_ipv4=172.16.222.13 ansible_ssh_host=172.16.222.13
+
+    [role=worker:vars]
+    consul_is_server=false
+    lvm_physical_device=/dev/sda3
+
+    [role=edge]
+    edge01 private_ipv4=172.16.222.16 ansible_ssh_host=172.16.222.16
+    edge02 private_ipv4=172.16.222.17 ansible_ssh_host=172.16.222.17
+
+    [role=edge:vars]
+    consul_is_server=false
+    lvm_physical_device=/dev/sda3
+
+    [dc=dc1]
+    control01
+    control02
+    control03
+    resource01
+    resource02
+    resource03
+    edge01
+    edge02
+
+I had to add the ansible_ssh_host line to run `playbooks/reboot-hosts.yml`
+
+The private_ipv4 is needed by several roles.
+
+The dc=dc1 group is needed to set `consul_dc_group` in the consul roles. And is specifically usd in this setup in the
+dnsmasq role.
+
+Note that dc1 is the default.  If you change the name of the data center in your inventory file
+you will need to set consul_dc in an environment variable.  For example, if you called your dc 'mydc' then you
+would need to enter:
+
+    ansible-playbook -u centos -i inventory -e consul_dc=mydc \
+            -e provider=bare-metal  -e @security.yml  terraform.sample.yml >& bare-metal.log
+
+the rest of the options will be discussed below.
+
+## Getting Started with Ansible
+
+
+Add your key to all the machines in your inventory
+
+    ansible all -i inventory  -u centos -k -m authorized_key -a "user=centos key=https://github.com/youraccount.keys"
+
+Note this makes use of your public key on github.  if you don't have a git hub account or a key pair on your git hub
+account, get one.
+
+the -k is needed cause I have to have it ask for a password at this point.
```

Comment:
It'd be best to avoid the first person here. Additionally, this is a little unclear: are you asking Ansible for a password, or is Ansible asking you for a password?